docs/sources/reference/run.md

```diff
@@ -460,6 +461,13 @@ division of CPU shares:
     101    {C1}		1	100% of CPU1
     102    {C1}		2	100% of CPU2
 
+The `--cpu-quota` flag limits the container's CPU usage. The default 0 value
+allows the container to take 100% of a CPU resource(1 CPU). The Completely Fair
+Scheduler(CFS) handles resource allocation for executing processes and is default
```

Comment:
You've reversed the order of the definition versus exposition you've used everywhere else - CFS (Completely Fair Scheduler)